cstrml/staking/src/tests.rs

```diff
@@ -301,13 +302,13 @@ fn rewards_should_work() {
             );
             assert_eq_error_rate!(
                 Balances::total_balance(&10) / 1000000,
-                (init_balance_10 + staking_reward * 500 / 2001) / 1000000,
+                (init_balance_10 + authoring_reward * 2 /3 + staking_reward * 500 / 2001) / 1000000,
```

Comment:
Why this failed `reward` modules?

Reply:
In the latest substrate's staking, ErasPointRewards uses BTreeMap to store point instead of vector. The key is the account ID instead of the original validator index. `reward_by_ids` changed the behavior. In the tests, authoring_reward is not related with election result anymore. It's just related with `reward_by_ids`. In the real chain environment, `reward_by_ids` is called according to the election result, which is correct. It's more like a decouple improvement.

Reply:
Change the tests back based on #250